### DISCUSSION TITLE
Did something break?

### DISCUSSION THREAD

**Discussion by ghost** [csstools/postcss-plugins#408] [General]
I created a new react project: npx create-react-app my-app.
Ran: npm install
Started app: npm start
Getting Error: Loading PostCSS "postcss-preset-env" plugin failed: Cannot find module 'node:vm'
![Capture](https://user-images.githubusercontent.com/7959372/171262003-4874dab6-a15b-44b0-af8b-2bd3946e0153.JPG)

**Comment by romainmenke** [comment]
A fix has already been merged and will be released shortly :

https://github.com/csstools/postcss-plugins/pull/407

You can downgrade to `postcss-preset-env` `v7.6.0` or update your node version to at least 16 to work around the issue until we publish the fix.
